code/modules/antagonists/eldritch_cult/knowledge.dm/flesh_lore.dm

```diff
@@ -0,0 +1,257 @@
+	gain_text = "Hundred's of us starved, but I.. I found the strength in my greed."
+	banned_knowledge = list(/datum/eldritch_knowledge/base_ash,/datum/eldritch_knowledge/base_rust,/datum/eldritch_knowledge/ash_final,/datum/eldritch_knowledge/rust_final)
+	next_knowledge = list(/datum/eldritch_knowledge/flesh_grasp)
+	required_atoms = list(/obj/item/kitchen/knife,/obj/effect/decal/cleanable/blood)
+	result_atoms = list(/obj/item/melee/sickly_blade/flesh)
+	cost = 1
+	route = "Flesh"
+
+/datum/eldritch_knowledge/flesh_ghoul
+	name = "Imperfect Ritual"
+	desc = "Allows you to resurrect the dead as voiceless dead by sacrificing them on the transmutation rune with a poppy. Voiceless dead are mute and have 50 HP. You can only have 2 at a time."
+	gain_text = "I found notes.. notes of a ritual, it was unfinished and yet i still did it."
+	cost = 1
+	required_atoms = list(/mob/living/carbon/human,/obj/item/reagent_containers/food/snacks/grown/poppy)
+	next_knowledge = list(/datum/eldritch_knowledge/flesh_mark,/datum/eldritch_knowledge/armor,/datum/eldritch_knowledge/ashen_eyes)
+	route = "Flesh"
+	var/max_amt = 2
+	var/current_amt = 0
+	var/list/ghouls = list()
+
+/datum/eldritch_knowledge/flesh_ghoul/on_finished_recipe(mob/living/user,list/atoms,loc)
+	var/mob/living/carbon/human/H
+	for(var/mob/living/carbon/human/humie in atoms)
+		H = humie
+	if(!H)
+		return
+	var/mob/dead/observer/G
+	for(var/mob/dead/observer/ghost in GLOB.dead_mob_list) //excludes new players
+		if(ghost.mind && ghost.mind.current == H && ghost.client)  //the dead mobs list can contain clientless mobs
+			G = ghost
+			break
+	if(!G.reenter_corpse())
+		var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a [H.real_name], a voiceless dead", ROLE_ECULTIST, null, ROLE_ECULTIST, 50,H)
+		if(!LAZYLEN(candidates))
+			return
+		var/mob/dead/observer/C = pick(candidates)
+		message_admins("[key_name_admin(C)] has taken control of ([key_name_admin(H)]) to replace an AFK player.")
+		H.ghostize(0)
+		H.key = C.key
+	if(!H.mind || !H.client)
+		to_chat(user, "<span class='warning'>There is no soul connected to this body...</span>")
+		return
+	if(!check_ghouls() || HAS_TRAIT(H,TRAIT_GHOUL))
+		return
+
+	ADD_TRAIT(H,TRAIT_MUTE,MAGIC_TRAIT)
+	ADD_TRAIT(H,TRAIT_GHOUL,MAGIC_TRAIT)
+	H.revive(full_heal = TRUE, admin_revive = TRUE)
+	H.setMaxHealth(50)
+	H.health = 50 // Voiceless dead are much tougher than ghouls
+	H.fully_replace_character_name(H.real_name,"Voiceless [H.real_name]")
+	to_chat(H, "<span class='userdanger'>You have been revived by </span><B>[user.real_name]!</B>")
+	to_chat(H, "<span class='userdanger'>[user.p_theyre(TRUE)] your master now, assist [user.p_them()] even if it costs you your new life!</span>")
+	atoms -= H
+	ghouls += H
+
+/datum/eldritch_knowledge/flesh_ghoul/proc/check_ghouls()
+
+	listclearnulls(ghouls)
+
+	for(var/mob/living/carbon/human/ghoul in ghouls)
+		if(ghoul.stat == DEAD)
+			ghouls -= ghoul
```

Comment:
me and the boys strangling and then defibbing each my new voiceless dead minions so that I can have 27 of them at once:
https://giphy.com/gifs/warnerarchive-classic-film-hammer-films-the-curse-of-frankenstein-3o85xuyd7tzEpDWQhi

Reply:
good catch, ill dust them when they die.